File: Examples/Features/Formatting/Fonts/FontStyle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formatting.Fonts
{
    public class FontStyle : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            IWorksheet worksheet = workbook.Worksheets[0];

            worksheet.Range["A1"].Value = "Bold";
            worksheet.Range["A1"].Font.Bold = true;

            worksheet.Range["A2"].Value = "Italic";
            worksheet.Range["A2"].Font.Italic = true;

            worksheet.Range["A3"].Value = "Bold Italic";
            worksheet.Range["A3"].Font.Bold = true;
            worksheet.Range["A3"].Font.Italic = true;
        }
    }
}

File: Examples.VB/Features/Formatting/Fonts/FontStyle.vb
Namespace Features.Formatting.Fonts
    Public Class FontStyle
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            Dim worksheet As IWorksheet = workbook.Worksheets(0)
            worksheet.Range!A1.Value = "Bold"
            worksheet.Range!A1.Font.Bold = True
            worksheet.Range!A2.Value = "Italic"
            worksheet.Range!A2.Font.Italic = True
            worksheet.Range!A3.Value = "Bold Italic"
            worksheet.Range!A3.Font.Bold = True
            worksheet.Range!A3.Font.Italic = True
        End Sub
    End Class
End Namespace
